internal/model/reports_type.go

```diff
@@ -0,0 +1,64 @@
+package model
+
+var ReportsTypes = []ReportsType{
+	ReportsTypeJournal,
+	ReportsTypeSchedule,
+	ReportsTypeAccountsReceivable,
+	ReportsTypeDebt,
+}
+
+type ReportsType int
+
+const (
+	ReportsTypeUnknown ReportsType = iota
+	ReportsTypeJournal
+	ReportsTypeSchedule
+	ReportsTypeAccountsReceivable
+	ReportsTypeDebt
+)
+
+func (i ReportsType) String() string {
+	return i.Key()
+}
+
+func (i ReportsType) Translation() string {
+	switch i {
+	case ReportsTypeJournal:
+		return "Journal"
+	case ReportsTypeSchedule:
+		return "Schedule"
+	case ReportsTypeAccountsReceivable:
+		return "Accounts Receivable"
+	case ReportsTypeDebt:
+		return "Debt"
+	default:
+		return ""
+	}
+}
+
+func (i ReportsType) Key() string {
```

Comment:
does the `String()` method of these things ever vary? I wonder if `Key()` is actually needed, or if it could be `String()` instead